app.js

```diff
@@ -16,7 +18,85 @@ var wrongThree = ('Wrong.');
 var wrongFour = ('Wrong, Isak has all of his toes.');
 var wrongFive = ('Wrong, Isak loves motorcycles.');
 
-//Questions 1-5
+var questionOne = ('Please answer with Y or N.  Does Isak have a dog?');
+var questionTwo = ('Please answer with Y or N.  Does Isak have a disease?');
+var questionThree = ('Please answer with Y or N. Does Isak brush his teeth every day?');
+var questionFour = ('Does Isak have all of his toes?');
+var questionFive = ('Does Isak ride motorcycles?');
+
+var askingQuestions = [questionOne, questionTwo, questionThree, questionFour, questionFive];
+var correctResponses = [wrongOne, wrongTwo, correctThree, correctFour, correctFive];
+var wrongResponses = [correctOne, correctTwo, wrongThree, wrongFour, wrongFive];
+var correctAnswers = ['N', 'N', 'Y', 'Y', 'Y'];
+
+var numberGuessCorrect = 'Wow, you actually got it!';
+var numberGuessHigh = 'Nope, too high.';
+var numberGuessLow = 'Nope, too low.';
+var questionSeven;
+
+var states = ['Nevada', 'Georgia', 'North Carolina'];
+var arrayLength = states.length;
```

Comment:
It's generally good to store things in readable variables, but in this case you've made it harder to read by switching to something less specific.